src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/config/TaskScheduler.kt

```diff
@@ -6,31 +6,34 @@ import org.springframework.scheduling.annotation.Scheduled
 import org.springframework.stereotype.Component
 import uk.gov.justice.digital.hmpps.pecs.jpc.tasks.BackfillReportsTask
 import uk.gov.justice.digital.hmpps.pecs.jpc.tasks.PreviousDaysLocationMappingTask
-import uk.gov.justice.digital.hmpps.pecs.jpc.tasks.PreviousDaysReportsTask
+import uk.gov.justice.digital.hmpps.pecs.jpc.tasks.PreviousDaysMovesTask
+import uk.gov.justice.digital.hmpps.pecs.jpc.tasks.PreviousDaysPeopleAndProfilesTask
 
 @ConditionalOnWebApplication
 @Component
 class TaskScheduler(
-  val previousDaysReportsTask: PreviousDaysReportsTask,
-  val previousDaysLocationMappingTask: PreviousDaysLocationMappingTask,
-  val backfillReportsTask: BackfillReportsTask
+  val previousDaysMoves: PreviousDaysMovesTask,
+  val previousDaysPeopleAndProfiles: PreviousDaysPeopleAndProfilesTask,
+  val previousDaysLocationMapping: PreviousDaysLocationMappingTask,
+  val backfillReports: BackfillReportsTask
 ) {
 
   @Scheduled(cron = "\${CRON_IMPORT_REPORTS}")
   @SchedulerLock(name = "importReports")
   fun previousDaysReportsTask() {
-    previousDaysReportsTask.execute()
+    previousDaysMoves.execute()
```

Comment:
So splitting out her for now to avoid the need to add another CRON job env var.  This may need to be scheduled to run with a bit of gap if it does not resolve the issue.